src/database/models/InterviewPrep/UserSheet.ts

```diff
@@ -0,0 +1,78 @@
+import { UserSheetModel } from '@/interfaces';
+import { Model, Schema, model, models } from 'mongoose';
+import { databaseModels } from '@/constant';
+
+const UserQuestionSchema = new Schema(
+  {
+    questionId: {
+      type: Schema.Types.ObjectId,
+      ref: databaseModels.INTERVIEW_SHEET_QUESTION,
```

Comment:
Don't refer Question here because Question is already nested inside INTERVIEW_SHEET. No need to ref at all